src/DependencyInjection/NexySlackExtension.php

```diff
@@ -26,11 +27,17 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
+        $guzzleService = $config['guzzle_service'];
+        unset($config['guzzle_service']);
```

Comment:
You can combine this with the unset below: `unset($config['guzzle_service'], $config['endpoint'])`